packages/services/api/src/modules/schema/module.graphql.ts

```diff
@@ -376,6 +376,10 @@ export default gql`
 
   input GitHubSchemaCheckInput {
     commit: String!
+    """
+    The repository name of the schema check.
+    """
+    repository: String
```

Comment:
Maybe we should also store the GitHub commit sha? That way we can even link to the exact commit from within Hive.

Reply:
yes, let's do it

Reply:
[`52899c0` (#2701)](https://github.com/kamilkisiela/graphql-hive/pull/2701/commits/52899c0e2d8db4b7254ad581f3c360a8d19eb232)